heroku/heroku-vscode
# Added dyno scaling support for Formations. Added a stub for the Add-ons Marketplace

This PR provides the following:

1. Added dyno scaling support for Formations. 
2. Added a stub for the Add-ons Marketplace. 
3. Added status polling for Dyno startup and restarts.

## To Test
1. Checkout this branch, install and build
2. Run the extension and open a Heroku app source code.
3. Verfy you have created at least 1 heroku remote for the app.
4. Verify you have at least 1 formation that allows scaling beyond 1 Dyno. 
5. Click on the edit icon next to your Formation name in the TreeView and update the number of dynos.
6. Observe the dynos in the TreeView updates and shows a spinning icon until the dyno has started.
